deployer/src/helpers.py

```diff
@@ -46,61 +41,64 @@ def make_request(endpoint, type=None, data=None, username=None, password=None,
                  json_request=False):
     import requests
 
-    url = base_url + endpoint if "://" not in endpoint else endpoint
+    if "://" not in endpoint:
+        print("Wrong endpoint:{}, it should be a complete URL".format(endpoint))
+        exit(6)
 
-    success_codes = [200, 201, 204]
+    if username is None or password is None:
+        print("Credentials not complete to call:{}, username and password should be set".format(endpoint))
```

Comment:
```suggestion
        print("{}: username and password are missing".format(endpoint))
```

Reply:
(same here: validation of the config should be done outside - but that is for another commit ;) )